src/hotspot/share/runtime/sharedRuntime.cpp

```diff
@@ -519,7 +519,7 @@ JRT_LEAF(jfloat, SharedRuntime::hf2f(jshort x))
   }
 
   // Add the bias of float exponent and shift
-  int float_exp_bits = (hf_exp + 127) << 24 - 1;
+  int float_exp_bits = ((hf_exp + 127) << 24 - 1);
```

Comment:
It will be more clear to have the parenthesis like this:
int float_exp_bits = (hf_exp + 127) << (24 - 1);

May be we should refer to src/java.base/share/classes/java/lang/Float.java:floatToFloat16 and Float16toFloat in comments.